nostalgia/sources/apple/__init__.py

```diff
@@ -0,0 +1,15 @@
+from nostalgia.ndf import NDF
+from nostalgia.sources.google import Google
+
+
+class Apple(NDF):
+    vendor = "apple"
+    ingest_settings = {
+        "ingest_glob": "~/Downloads/apple/*.zip",
```

Comment:
Again, do we want to change this to `~/nostalgia_data/input/apple/*.zip`?

Reply:
Actually, the ingest glob is there to help to pick it up automatically after being downloaded. It can even unpack zips. 

See the `load_from_download` function in /nostalgia/extracter.py

So it is probably near correct, I am only concerned about `apple/` - how did they get into a folder?

Reply:
Yes of course about paths, it will be changed into something generic, @jjc385, but thanks

@kootenpv on the privacy.apple.com they give links to each particular archive, so I moved all them to `apple` folder to work on.
Otherwise we can't use wildcard to take all data ingested.

Reply:
What are the names of the files in it? Maybe there is still a chance.

Also, I thought perhaps we could scan the download folder for zip files that when we look at the archive info, is actually the structure we expect?

Reply:
I would say it's easier to make a folder, where you can take all archives and ingest them.
Otherwise we should potentially have a list of all these archives? I can't say now that it will never change.